app/scripts/extractors/pdf_extractor.py

```
@@ -0,0 +1,403 @@
+
+def _extract_tables(pdf_path: Path, table_dir: Path) -> tuple[list[TableItem], list[str]]:
+    issues: list[str] = []
+    out: list[TableItem] = []
+    if camelot is None:
+        issues.append("camelot_not_installed")
+        return out, issues
+
+    try:
+        try:
+            tables = camelot.read_pdf(str(pdf_path), pages="all", flavor="lattice")
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)